chapters/operatorsandexpressions.tex

```diff
@@ -393,29 +393,34 @@ \subsection{Built-in Variable time}\label{built-in-variable-time}\indexinline{ti
 \end{lstlisting}
 \end{example}
 
-\section{Built-in Intrinsic Operators with Function Syntax}\label{built-in-intrinsic-operators-with-function-syntax}
+\section{Built-in Operators and Functions}\label{built-in-intrinsic-operators-with-function-syntax}
 
-Certain built-in operators of Modelica have the same syntax as a function call.
+Certain built-in operators of Modelica are called using the same syntax as a function call.
 However, they do not behave as a mathematical function, because the result depends not only on the input arguments but also on the status of the simulation.
 
 There are also built-in functions that depend only on the input argument, but also may trigger events in addition to returning a value.
-Intrinsic means that they are defined at the Modelica language level, not in the Modelica library.
-The following built-in intrinsic operators/functions are available:
+The built-in functions may also be overloaded such that a single Modelica function cannot be compatible with all calls of the function.
+Built-in means that they are defined at the Modelica language level, not in the Modelica library.
+The following built-in operators/functions are available:
 \begin{itemize}
 \item
-  Mathematical functions and conversion functions, see \cref{numeric-functions-and-conversion-functions} below.
+  Mathematical functions and conversion operators, see \cref{numeric-functions-and-conversion-functions} below.
 \item
   Derivative and special purpose operators with function syntax, see \cref{derivative-and-special-purpose-operators-with-function-syntax} below.
 \item
-  Event-related operators with function syntax, see \cref{event-related-operators-with-function-syntax} below.
+  Event-related functions, see \cref{event-related-operators-with-function-syntax} below.
```

Comment:
But the event-generating stuff is not following the variability rules of normal function calls, so they should be operators, right?
```suggestion
  Event-related operators, see \cref{event-related-operators-with-function-syntax} below.
```

Reply:
After we added GenerateEvents-annotations I think they can be seen as following the rules.
(With the caveat that the rules are slightly unclear.)

Reply:
Yeah, we haven't figured out those variability rules yet…